src/coffee/connect/rest.coffee

```diff
@@ -119,7 +119,15 @@ class Rest
   # Throws an {Error} if `access_token` could not be retrieved
   _preRequest: (params, callback) ->
     _req = (retry) =>
-      unless @_options.access_token
+      isTokenExpired = false
+      if @_tokenExpiration
+        expirationTime = Date.now() + (@_tokenExpiration * 1000) -
```

Comment:
There you go https://github.com/sphereio/sphere-node-sdk/pull/127/commits/ad1e2c6ffb069fbd820eda9d19f6f8fff57d2c26